website/docs/r/kms_instance_policies.html.markdown

```diff
@@ -12,7 +12,9 @@ description: |-
 Provides a resource to manage instance policies for Key Protect and Hyper Protect Crypto Service (HPCS) services. This allows instance policies to be created and updated. Instance policies can be created for an existing kms instance resource.
 
 **NOTE**
-: `terraform destroy` does not remove the policies of the Instance but only clears the state file. Instance Policies get deleted when the associated Instance resource is destroyed.
+- `terraform destroy` does not remove the policies of the Instance but only clears the state file. Instance Policies get deleted when the associated Instance resource is destroyed.
+
+- Terraform sets the value `0` for `interval_month` in the state file when no value is passed in the input because it is the default value for the `int` data type. So, the value `0` for `interval_month` in the state file indicates that the `interval_month` is not set.
 
```

Comment:
The value `0` for `interval_month` in the `tfstate` file indicates that the `interval_month` is not set in the input. Terraform sets this value to `0` in the `tfstate` file because `0` is the default value for the `int` data type of this field.